CODE_OF_CONDUCT.md

```diff
@@ -3,7 +3,7 @@
 ## Our Pledge
 
 In the interest of fostering an open and welcoming environment, we as
-contributors and maintainers pledge to making participation in our project and
```

Comment:
@NinjaCoderDev, I would not make changes to this file as it's a standard template for Adobe open source projects. The originals in this file are actually okay.